### PR TITLE
Add more information about migrating for various commands

### PR DESCRIPTION
**Pull request by prince-chrismc**

### CODE REVIEW

File: migrating_to_2.0/commands.rst [hunk 1]
```diff
@@ -77,6 +86,18 @@ This is the substitute of the old "conan user".
      $ conan remote login [-h] [-f FORMAT] [-v [V]] [--logger] [-p [PASSWORD]] remote username
 
 
+conan upload
+^^^^^^^^^^^^
+
+The default behavoir has changed from requiring `--all` to include the binary packages to `--recipe-only` for just the recipe
```
Comment (memsharded) [inline migrating_to_2.0/commands.rst:92]:
```suggestion
The default behavior has changed from requiring `--all` to include the binary packages to `--recipe-only` for just the recipe 
```

File: migrating_to_2.0/commands.rst [hunk 2]
```diff
@@ -115,24 +136,36 @@ There are some examples:
...
 
+Removed "conan user"
+^^^^^^^^^^^^^^^^^^^^
+
+This has been replaced with :ref:<conan_v2_remote_login>
+
+Removed "conan config set"
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+we are no longer implmenting file-editing commands in 2.0. A bit overkill `conan config set` to edit one file. Which should very rarely happen,
```
Comment (memsharded) [inline migrating_to_2.0/commands.rst:166]:
```suggestion
we are no longer implementing file-editing commands in 2.0. A bit overkill `conan config set` to edit one file. Which should very rarely happen,
```

**Review by memsharded — APPROVED** [review]

File: migrating_to_2.0/commands.rst [hunk 3]
```diff
@@ -115,24 +136,36 @@ There are some examples:
...
+Removed "conan user"
+^^^^^^^^^^^^^^^^^^^^
+
+This has been replaced with :ref:<conan_v2_remote_login>
+
+Removed "conan config set"
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+we are no longer implementing file-editing commands in 2.0. A bit overkill `conan config set` to edit one file. Which should very rarely happen,
+the file is updated with `conan config install`.
```
Comment (SSE4) [inline migrating_to_2.0/commands.rst:167]: you can pass config commands with `-c` flag, it could be a reasonable replacement for most of the use-cases

Comment (prince-chrismc) [reply migrating_to_2.0/commands.rst:167]:
```suggestion
the file is updated with `conan config install`. Alternatively, you can use the command line and profiles to pass these values.
```

**Pull request merged**